tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "lib": ["es5", "es2015.collection", "dom"],
-    "types": [],
+    "types": ["jest"],
```

Comment:
This was necessary for WebStorm to interpret test files correctly. Opened the project with VSCode as well to make sure nothing was broken.

Reply:
Removed in latest commit after offline conversation, since this would affect non-test files as well.